site/code/index.html.js

```diff
@@ -17,41 +17,36 @@ var Code = React.createClass({
 
         <section className="content documentationContent nosidebar">
           <div className="inner-content">
-            <h1>Releases</h1>
+            <h1>Code</h1>
             <Marked>{`
 
-GraphQL consists of the following open-source repos on [Github](https://github.com/graphql/).
+Many different programming languages support GraphQL. This list contains some of the more popular server-side frameworks, client libraries, and other useful stuff.
 
-## Official
+## GraphQL Server Libraries
 
-### Spec
+  - [GraphQL.js](/graphql-js/) ([github](https://github.com/graphql/graphql-js)) ([npm](https://www.npmjs.com/package/graphql)): The reference implementation of the GraphQL specification, designed for running a GraphQL server in a Node.js environment.
+  - [express-graphql](/graphql-js/) ([github](https://github.com/graphql/express-graphql)) ([npm](https://www.npmjs.com/package/express-graphql)): The reference implementation of a GraphQL API server over an Express webserver. You can use this to run GraphQL in conjunction with a regular Express webserver, or as a standalone GraphQL server.
+  - [Graphene](http://graphene-python.org/) ([github](https://github.com/graphql-python/graphene)): A Python library for building GraphQL APIs. Built-in support for [Relay](https://facebook.github.io/relay/), Django, SQLAlchemy, and Google App Engine.
+  - [graphql-ruby](https://github.com/rmosolgo/graphql-ruby): A Ruby library for building GraphQL APIs. Built-in support for Relay and Rails.
+  - [graphql-java](https://github.com/graphql-java/graphql-java): A Java library for building GraphQL APIs.
+  - [graphql-relay](https://github.com/graphql/graphql-relay-js) ([npm](https://www.npmjs.com/package/graphql-relay)): A server library to help construct a Node.js GraphQL API that supports [Relay](https://facebook.github.io/relay/).
+  - [Sangria](http://sangria-graphql.org/) ([github](https://github.com/sangria-graphql/sangria)) A Scala GraphQL library that supports [Relay](https://facebook.github.io/relay/).
+  - [Apollo Server](http://dev.apollodata.com/tools/apollo-server/index.html) ([github](https://github.com/apollostack/apollo-server)) ([npm](https://www.npmjs.com/package/apollo-server)): A GraphQL server that works with all Node.js HTTP server frameworks: Express, Connect, HAPI and Koa. Designed to work well with the Apollo Client.
 
-  - **[graphql](https://github.com/facebook/graphql)** ([html](http://facebook.github.io/graphql/)): A working draft of the specification for GraphQL.
+## GraphQL Clients
 
-### Server (JavaScript)
+  - [Relay](https://facebook.github.io/relay/) ([github](https://github.com/facebook/relay)) ([npm](https://www.npmjs.com/package/react-relay)): Facebook's framework for building React applications that talk to a GraphQL backend.
+  - [Apollo Client](http://dev.apollodata.com/react/) ([github](https://github.com/apollostack/apollo-client)): A powerful JavaScript GraphQL client, designed to work well with the Apollo Server.
```

Comment:
Apollo client works equally well with every GraphQL server, and doesn't require any special features at all. So let's remove that. I'd also emphasize that it works with React and Angular 2.